site/src/pages/DeploySettingsPage/Option.tsx

```diff
@@ -1,5 +1,5 @@
 import { css, type Interpolation, type Theme, useTheme } from "@emotion/react";
-import CheckCircleOutlined from "@mui/icons-material/CheckCircleOutlined";
+import LabelImportant from "@mui/icons-material/LabelImportant";
```

Comment:
Nit: when importing MUI icons, I usually like to add `Icon` to the name of the import. Especially here, when I see the word "label" with no other qualifiers, I expect something based on the HTML `<label>` element